vm/vm.go

```diff
@@ -1171,7 +1171,12 @@ func (vm *VM) loadAcceptedBlocks(ctx context.Context) error {
 		start = vm.lastAccepted.Hght - lookback
 	}
 	for i := start; i <= vm.lastAccepted.Hght; i++ {
-		blk, err := vm.GetDiskBlock(ctx, i)
+		blkID, err := vm.GetBlockIDAtHeight(ctx, i)
+		if err != nil {
+			vm.snowCtx.Log.Info("could not find block ID on-disk", zap.Uint64("height", i))
+			continue
+		}
+		blk, err := vm.GetStatelessBlock(ctx, blkID)
```

Comment:
This code should be unchanged, this is called on startup when there are no blocks in the cache, so we need to load all of these from disk and can call `GetDiskBlock` directly

Reply:
Thanks, done!